test/integration.js

```diff
@@ -740,5 +752,99 @@ tape('targetingRules: multiple rules are applied, pricingBounds are honored', as
 	t.end()
 })
 
+tape('validatorWorker: does not apply empty aggr', async function(t) {
```

Comment:
@Ivshti here is the test that checks for that the validatorWorker doesn't apply an empty aggr